package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "server.js",
+  "dependencies": {
+    "add": "^2.0.6",
+    "babel-jest": "^27.3.1",
+    "jest": "^27.3.1",
+    "ts-jest": "^27.0.7"
+  },
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.25.3",
+    "hint": "^6.1.9",
+    "html-webpack-plugin": "^5.5.0",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack-cli": "^4.9.1"
+  },
+  "scripts": {
+    "test": "jest",
+    "start": "node server.js",
+    "build": "webpack"
+  },
```

Comment:
I am unable to use the `npm start` command as it gives an error. Please update `scripts` as:
`
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
`.

Reply:
Thank you @SajjadAhmad14 I have worked on all the request changes!